linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_suid_auid_privilege_function/tests/use_f_key.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# packages = audit
+
+OTHER_FILTERS=" -C euid!=uid -F auid!=unset"
+
+echo "-a always,exit -F arch=b32 -S execve${OTHER_FILTERS_EUID} -F key=user_emulation" > /etc/audit/rules.d/privileged.rules
+echo "-a always,exit -F arch=b64 -S execve${OTHER_FILTERS_EUID} -F key=user_emulation" >> /etc/audit/rules.d/privileged.rules
```

Comment:
A wrong variable name cause this test to fail.

```suggestion
echo "-a always,exit -F arch=b32 -S execve${OTHER_FILTERS} -F key=user_emulation" > /etc/audit/rules.d/privileged.rules
echo "-a always,exit -F arch=b64 -S execve${OTHER_FILTERS} -F key=user_emulation" >> /etc/audit/rules.d/privileged.rules
```

Reply:
> A wrong variable name cause this test to fail.

Thank you!
The changes have been addressed.